tests/dummy/app/styles/app.css

```diff
@@ -0,0 +1,3 @@
+ul li {
+  height: 500px;
+}
```

Comment:
This style was necessary in order to prevent multiple "pages" from loading in acceptance tests, before the first `andThen` callback was called